build/push

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+IMAGEID=$( docker images -q litmuschaos/py-runner)
+
+if [ ! -z "${DNAME}" ] && [ ! -z "${DPASS}" ];
+then
+  docker login -u "${DNAME}" -p "${DPASS}";
+  #Push to docker hub repository with latest tag
+  docker tag ${IMAGEID} litmuschaos/py-runner:ci
+  docker push litmuschaos/py-runner:ci;
```

Comment:
we need to push release tag also pls refer: https://github.com/litmuschaos/litmus-go/blob/master/build/push